BlueprintUI/Sources/Layout/ConstrainedAspectRatio.swift

```diff
@@ -85,12 +166,11 @@ public extension Element {
     ///
     /// - parameters:
     ///   - aspectRatio: The aspect ratio that the content size should match.
-    ///   - contentMode: Whether the aspect ratio should be reached by expanding the content
-    ///     element's size to fill its parent or shrinking it to fit.
+    ///   - contentMode: How the content should size itself relative to its parent.
     ///
     func constrainedTo(
         aspectRatio: AspectRatio,
-        contentMode: ConstrainedAspectRatio.ContentMode = .fill
+        contentMode: ConstrainedAspectRatio.ContentMode = .fitContent
```

Comment:
This doesn't change the default behaviour, right?

Reply:
Correct!